dapp/src/components/common/FeaturedCommunity/index.jsx

```diff
@@ -1,16 +1,52 @@
-import React, { memo } from 'react'
+import React, { memo, useEffect } from 'react'
+import { connect } from 'react-redux'
+import { fetchToken } from 'actions/token'
+import { fetchMetadata } from 'actions/metadata'
+import { balanceOfToken } from 'actions/accounts'
+
 import CommunityPlaceholderImage from 'images/community_placeholder.png'
 
 const FeaturedCommunity = memo(({
+  accountAddress,
   community,
   showDashboard,
   metadata
 }) => {
+  const {
+    communityURI,
+    homeTokenAddress,
+    foreignTokenAddress
+  } = community
+
+  useEffect(() => {
+    if (accountAddress) {
```

Comment:
this will fetch on every render? or  not because of the `memo`? maybe it makes sense to do:
``use effect(() => {... }, [accountAddress])